json/schema/siglist-01.json

```diff
@@ -22,10 +22,22 @@
                     "inputs": {
                         "type": "array",
                         "items": {
-                            "type": "array",
-                            "items": {
-                                "type": "string",
-                                "description": "Hex-encoded ECDSA signature"
+                            "type": "object",
+                            "properties": {
+                                "required": {
+                                    "type": "integer",
+                                    "description": "Number of signatures remaining before the input is fully signed",
```

Comment:
It's not clear to me if the value of required is a constant always equal to the reqSigs parameter of the input's series or if it is the difference between the series' reqSigs and the number of signatures in the array.